src/routes/settings/(nav)/assistants/[assistantId]/+page.svelte

```diff
@@ -147,11 +162,48 @@
 		</div>
 	</div>
 
-	<h2 class="mt-4 text-lg font-semibold">System Instructions</h2>
+	<!-- two columns for big screen, single column for small screen -->
+	<div class="mb-12 mt-3">
+		<h2 class="mb-2 font-semibold">System Instructions</h2>
+		<textarea
+			disabled
+			class="box-border h-full min-h-[8lh] w-full rounded-lg border-2 border-gray-200 bg-gray-100 p-2 disabled:cursor-not-allowed"
+			>{assistant?.preprompt}</textarea
+		>
 
-	<textarea
-		disabled
-		class="min-h-[8lh] w-full flex-1 rounded-lg border-2 border-gray-200 bg-gray-100 p-2 disabled:cursor-not-allowed 2xl:min-h-[12lh]"
-		>{assistant?.preprompt}</textarea
-	>
+		{#if hasRag}
+			<div class="mt-4">
+				<h2 class=" font-semibold">Internet Access</h2>
+				{#if assistant?.rag?.allowAllDomains}
+					<p class="text-sm text-gray-500">
+						This Assistant uses the Web Search to find information on Internet.
```

Comment:
```suggestion
						This Assistant uses Web Search to find information on Internet.
```
to be consistent with "This Assistant can use Web Search on the following domains:", no "the"